documentation/asciidoc/computers/software-sources.adoc

```diff
@@ -153,6 +153,9 @@ A tool to provision Raspberry Pi devices at manufacture, including secure boot a
 https://github.com/raspberrypi/usbboot::
 A tool to boot a Raspberry Pi over USB for provisioning Compute Module and Raspberry Pi devices.
 
+https://github.com/raspberrypi/rpi-sb-provisioner::
+A tool to mass provision Raspberry Pi Compute Modules (Compute Module 4 and later are supported).
```

Comment:
The README in that repo says "for Raspberry Pi Devices", rather than restricting it just to Compute Modules... :thinking: